src/main/java/org/fundacionjala/trello/config/Environment.java

```diff
@@ -0,0 +1,139 @@
+    private static final String ACCOUNTS = "accounts";
+    private static final String USER_NAME = "user";
+    private static final String ROOT_PATH = ".";
+    private static final String IMPLICIT_TIME_WAIT = "implicitTimeWait";
+    private static final String EXPLICIT_TIME_WAIT = "explicitTimeWait";
+    private static Environment instance;
+
+    private final Properties properties;
+    private Map<String, Object> envConfig;
+
+    /**
+     * Initializes instance of Properties class.
+     */
+    private Environment() {
+        List<Map<String, Object>> environments;
+        try (FileInputStream fileInputStream = new FileInputStream(PROPERTIES_FILE_PATH)) {
+            properties = new Properties();
+            properties.load(fileInputStream);
+            environments = JsonPath.with(Files.readString(Paths.get(JSON_CONFIG_FILE_PATH), StandardCharsets.UTF_8))
+                    .getList(ROOT_PATH);
+        } catch (FileNotFoundException e) {
+            throw new RuntimeException("Environment config file not found");
+        } catch (IOException e) {
+            throw new RuntimeException("Input/Output exception, failed to load gradle.properties");
+        }
+        envConfig = environments.stream().filter(env -> env.get(ENVIRONMENT_NAME).equals(getEnvironmentName()))
+                .findFirst().orElse(new HashMap<>());
+    }
+
+    /**
+     * Gets singleton instance of the EnvironmentReader class.
+     *
+     * @return EnvironmentReader instance.
+     */
+    public static Environment getInstance() {
+        if (instance == null) {
+            instance = new Environment();
+        }
+        return instance;
+    }
+
+    /**
+     * Gets string containing a specific environment property.
+     *
+     * @param env is the property name string.
+     * @return the wanted property value.
+     */
+    private String getEnvProperty(final String env) {
+        String property = System.getProperty(env);
+        if (property == null) {
+            return properties.getProperty(env);
+        }
+        return property;
+    }
+
+    /**
+     * Gets environment name property.
+     *
+     * @return environment name.
+     */
+    public String getEnvironmentName() {
+        return getEnvProperty(ENVIRONMENT);
+    }
+
+    /**
+     * Gets environment base UI URL.
+     *
+     * @return environment base UI URL.
+     */
+    public String getUiBaseUrl() {
+        return envConfig.get(UI_BASE_URL).toString();
+    }
+
+    /**
+     * Gets environment base API URI.
+     *
+     * @return environment base API URI.
+     */
+    public String getApiBaseUri() {
+        return envConfig.get(API_BASE_URI).toString();
+    }
+
+    /**
+     * Gets account from environment config.
+     *
+     * @param username account username.
+     * @return account json path object.
+     */
+    @SuppressWarnings("unchecked")
+    public Map<String, String> getAccount(final String username) {
```

Comment:
I suggest you change "username" variable to only "user" and "USER_NAME" to "USER" 
It's seems how it's looking for the username but the value of USER_NAME is equals to "user"